packages/destination-actions/src/destinations/hubspot/sendCustomBehavioralEvent/index.ts

```diff
@@ -67,7 +67,7 @@ const action: ActionDefinition<Settings, Payload> = {
       defaultObjectUI: 'keyvalue:only'
     }
   },
-  perform: (request, { payload }) => {
+  perform: async (request, { payload, settings }) => {
```

Comment:
nit: You can remove async given that we aren't handling promises here.
```suggestion
  perform: (request, { payload, settings }) => {
```